Problem_1.py

```diff
@@ -0,0 +1,25 @@
+# Time Complexity : O(n^2)
+# Space Complexity : constant
+
+class Problem_1:
+	def findContinuousArray(self, arr, k):
+		count = 0
+		for i in range(len(arr)):
+			temp_sum = arr[i]
+			if temp_sum == k:
+				count += 1
+				continue
+			for j in range(i + 1, len(arr)):
+				temp_sum += arr[j]
+				if temp_sum == k:
+					count += 1
+					break
+				elif temp_sum > k:
```

Comment:
you cannot use break here because the array values can be negative as well

Reply:
But if the sum is equal to the target, I can break the loop right